backend/src/main/config/config.js

```diff
@@ -0,0 +1,25 @@
+require('dotenv').config();
```

Comment:
is this required?

Reply:
Yes, in the future we will add our own env variables